restapi/stories.go

```diff
@@ -232,6 +235,22 @@ func (api api) DeleteStoryByID(w http.ResponseWriter, r *http.Request) render.Re
 		return rest.ErrInternal(api.logger, err)
 	}
 
+	imageKey, err := convertImageURLToKeyName(story.ImageURL)
```

Comment:
Are we confident that two stories wont share the same image? I remember you mention that bacblaze uses the same image if the filename is the same?

Reply:
If we're not sure I would suggest we add a guard to validate that story.ImageURL is unique (i.e only appears in one story) before deleting